pattern-library/sass/utilities/_afontgarde.scss

```diff
@@ -0,0 +1,57 @@
+/*! afontgarde - v0.1.6 - 2015-03-13
```

Comment:
Yeah, I did the same with ``_syntax.scss`` borrowed code used for code syntax styling in the Doc Site. Its not worth keeping things as ``.css`` files in a static vendor dir if they're not a bower package IMHO. I'll note in the README that A Font Garde is another tool we're using.